BUILD.gn

```diff
@@ -138,8 +136,17 @@ if (is_android) {
       "//brave/android/java/org/chromium/chrome/browser/shields/disconnect_entitylist.json",
       "//brave/android/java/org/chromium/chrome/browser/vpn/brave_vpn_confirm.json",
     ]
+    disable_compression = false
+  }
+
+  android_assets("brave_pak_assets") {
+    sources = [
+      "$root_out_dir/brave_100_percent.pak",
+      "$root_out_dir/brave_resources.pak",
+    ]
 
     deps = [
+      ":brave_compressable_assets",
       "//brave:browser_dependencies",
```

Comment:
I know this isn't part of this PR, but is this dep necessary? It doesn't seem like it should be.